rhoas-cli-kafka/README.adoc

```diff
@@ -0,0 +1,265 @@
+////
+START GENERATED ATTRIBUTES
+WARNING: This content is generated by running npm --prefix .build run generate:attributes
+////
+
+
+:community:
+:imagesdir: ./images
+:product-version: 1
+:product-long: Application Services
+:product: App Services
+:registry-product-long: OpenShift Service Registry
+:registry: Service Registry
+// Placeholder URL, when we get a HOST UI for the service we can put it here properly
+:service-url: https://console.redhat.com/beta/application-services/streams/
+:registry-url: https://console.redhat.com/beta/application-services/service-registry/
+:property-file-name: app-services.properties
+:rhoas-version: 0.29.0
```

Comment:
We may aswell update this now to the latest release.

Reply:
I never get why we doing it. This version is just informational right?

Reply:
There's no real need, there was some discussion in https://github.com/redhat-developer/app-services-cli/issues/826#issuecomment-895524573 around removing it.